CoreScriptsRoot/Modules/Panel3D.lua

```diff
@@ -85,13 +88,20 @@ function Panel3D.Get(panelId)
 
 		panel.transparencyCallbacks = {}
 
+		panel.OnMouseEnter = false
+		panel.OnMouseLeave = false
```

Comment:
Why are we settings these to false and not nil? I am getting nervous about the implications.